charts/sessionspaces/templates/clusterrole.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ include "sessionspaces.fullname" . }}
+rules:
+- apiGroups: [""]
+  resources: ["pods", "services"]
+  verbs: ["get", "list", "watch", "patch", "create"] 
+- apiGroups: [""]
+  resources: ["namespaces"]
+  verbs: ["create", "delete", "get", "list"]
+- apiGroups: [""]
+  resources: ["pods/exec"]
+  verbs: ["create"]
+- apiGroups: [""]
+  resources: ["pods/logs"]
+  verbs: ["get", "watch"]
+- apiGroups: ["rbac.authorization.k8s.io"]
+  resources: ["clusterroles"]
+  verbs: ["create", "patch"]
+- apiGroups: [""]
+  resources: ["configmaps"]
+  verbs: ["create", "patch"]
+- apiGroups: ["argoproj.io"]
+  resources: [
+    "workflowartifactgctasks",
+    "workflowartifactgctasks/status",
+    "workflowtaskresults",
+    "workflowtasksets",
+    "workflowtasksets/status",
+    "clusterworkflowtemplates",
+    "cronworkflows",
+    "eventsources",
+    "sensors",
+    "workfloweventbindings",
+    "workflows",
+    "workflowtemplates"
+  ]
+  verbs: ["get", "list", "watch", "patch", "create"]
```

Comment:
Sessionspaces shouldn't do anything with argo workflows resources

Reply:
It creates two clusterroles (visit-member, argo-workflows) with these resources.